### ISSUE TITLE
Stone axe head vs Metal axe head 

### ISSUE THREAD

**Issue by NyarlathotepZERO** [CleverRaven/Cataclysm-DDA#47129]
# Is your feature request related to a problem? Please describe.
Metal axe head is able to be crafted prior to a Stone axe head

# Describe the solution you'd like
By comparing their very description and statistics, I would imagine that a stone axe head would be achievable prior to the metal axe head. The stone description hints that it is shaped in a way that is useful for cutting wood, but the metal description hints that it has been worked in a way to make it useful for cutting wood.

# Describe alternatives you've considered
I would simply swap the skill requirements between the two to make stone achievable before the metal. 

# Additional context
The statistics of both in comparison clearly show the metal to be a superior device, and the stage of crafting each should reflect this.

**Comment by anothersimulacrum** [comment]
It makes total sense to me that a metal axe head is better than and easier to make than a stone one. The stone one is there for when you cannot get metal.

**Comment by NyarlathotepZERO** [comment]
> It makes total sense to me that a metal axe head is better than and easier to make than a stone one. The stone one is there for when you cannot get metal.

Yep, it makes total sense to me as well in regards to the metal being a superior device. Although I'd like you to explain your reasoning as to how metal is far easier to 'work' into an edge than using stone.

**Comment by anothersimulacrum** [comment]
I guess I don't really have any solid reasoning, it just seems like I'd be more able to figure out how to turn a hunk of metal into a tool than a hunk of stone.

**Comment by actual-nh** [comment]
This would seem to depend on exactly how good an axe you want to make. See #47111 for more on stone knapping; the more advanced stone axes would require lots of skill (just as does being a blacksmith!) but relatively crude versions are easier (unless your hunk of metal is already close to sharp, of course!). Hand axes came before handles, actually - the variety probably first used for butchering didn't have a handle.

**Comment by NyarlathotepZERO** [comment]
Yep, not talking about the axes with haft, I'm talking about the most primitive form (eg: little more than a hunk of steel/stone). The ones that require survival 4 (for metal) and survival 5 (for stone) to craft.

stone axe head: https://cdda-trunk.chezzo.com/hand_axe
and
metal axe head: https://cdda-trunk.chezzo.com/makeshift_axe

**Comment by Shadefang** [comment]
> Although I'd like you to explain your reasoning as to how metal is far easier to 'work' into an edge than using stone.

Note that I don't have data or studies to back this up, I'm just going off my own minor personal experience. Assuming you're starting from roughly the same spot (a rock vs a chunk of metal) the stone might take less physical effort to shape, or be less reliant on decent tools, but errors made when shaping stone are more difficult to avoid and recover from. The stone shaping is entirely material removal, breaking off chunks that you can't really reattach, whereas shaping metal (in this case) is almost entirely deforming the part. There are definitely limits to shaping metal in that way, but they're generally more forgiving than accurately breaking a rock.

My personal thoughts here are that the difficulties are roughly in the right place relative to each other (though possibly too heavily weighted into survival vs fabrication) and the only change I'd make is to add a requirement for "anvil 1" to the metal axe head. It's a logical requirement, and not too difficult to get through finding a large rock ( https://cdda-trunk.chezzo.com/qualities/ANVIL )

**Comment by actual-nh** [comment]
> Assuming you're starting from roughly the same spot (a rock vs a chunk of metal) the stone might take less physical effort to shape, or be less reliant on decent tools, but errors made when shaping stone are more difficult to avoid and recover from.

I can see this as being a problem for anything elaborate - but if you're constructing something crude, you can just start over again without much of a setback.

**Comment by NyarlathotepZERO** [comment]
These are all good points, and aren't too far from my own thoughts.

I cant help but think that if someone was wandering about in a CDDA environment scrounging in bushes for useful things. For every hunk of metal material they came across that was even remotely useful for cutting down a tree, they would probably have walked past 30 odd stones that would do the same job, albeit a little less efficiently.

Due to the superior properties of metal, given the two, I'd honestly take a metal axe over a stone one any day (when largely considering it's cutting ability). But as far as getting myself into a more comfortable survival situation asap, I certainly wouldn't be holding my breath whilst I found a piece of metal of appropriate proportions, and then tried working it into a shape that would be more useful than the 30 odd suitable pieces of stone I had walked past earlier. 

This is why I suggest that the stone be moved to a skill 'gamestage' that is more appropriate and accessible. This would define it more as a primitive tool, and give a nice space to allow a slight upgrade to a metal one. 

You are right in that in some light, working metal could be more forgiving than stone (especially in the case of an untrained hand) but as someone who has over 20 years experience on the bench in the metalsmithing trade, I can honestly say that overworked metal has a plethora of it's own drawbacks.

tl;dr give the 2 axes a chance to define their own spaces in the crafting progression tree

**Comment by Shadefang** [comment]
> I can see this as being a problem for anything elaborate - but if you're constructing something crude, you can just start over again without much of a setback.

I largely agree, but starting again in this case would probably also involve finding a new rock. I suppose I must amend my previous statement: I find the current level difference to be fine when it comes to difficulty of successfully crafting the tool, but it would make sense to be able to *attempt* to craft both of them at a lower level. I don't know enough about the way the game handles learning recipes to say how doable that is.

**Comment by actual-nh** [comment]
> I largely agree, but starting again in this case would probably also involve finding a new rock. I suppose I must amend my previous statement: I find the current level difference to be fine when it comes to difficulty of successfully crafting the tool, but it would make sense to be able to _attempt_ to craft both of them at a lower level. I don't know enough about the way the game handles learning recipes to say how doable that is.

I'm in a bit of a rush, but I _think_ that would be via decreasing the required skills for the stone axe head, but increasing the difficulty.

**Comment by NyarlathotepZERO** [comment]
I forgot to mention that if the idea was to work your metal to have it end up in any form that would be even half decent for cutting down a tree, it would be difficult to work it with anything less than another piece of metal. Your striking surface would also want to be metal to avoid losing much of your effort, manifested by it shattering and crumbling under repeated impact. A 'boulder anvil' would probably do in a pinch, but only as a *very* crude alternative.

Metal has a tendency to harden as it is worked (which is why it is constantly annealed throughout he process). Throwing it into a fire would suffice as a relatively crude attempt at achieving this. without hardening/quenching, the edge would only become more brittle the more time you spent working on it.

All this is related to forging though, if the idea was to instead use only a grindstone to hone an edge, then it would just straight up be more difficult and time consuming than fashioning one out of stone, even just considering the pure act of performing the ritual of grinding down the 2 different mediums. 

I have noticed that many tasks have critical failures when attempting it whilst underskilled etc, and perhaps using stone should utilize this somehow. Due to stone being more readily available something like this would probably bring it infinitely closer to a more realistic situation of "oh damn, I just cleaved my stone in half... throw it in the pile and start again".

**Comment by Shadefang** [comment]
> I forgot to mention that if the idea was to work your metal to have it end up in any form that would be even half decent for cutting down a tree, it would be difficult to work it with anything less than another piece of metal... ...very crude alternative.

Which I think fits well with the description:
  "This is a chunk of steel with one edge hammered down to something resembling a cutting edge.  It works passably well as an axe but really can't compare to a proper axe."
It's not proper forging, it's nor a proper axe head, it's using whatever you have to beat a lump of steel into something vaguely resembling an axe. As for the concerns about the edge ending up brittle, do you think that is as much of a major concern given the end result is intended to be roughly equivalent to a stone axe head?

>I have noticed that many tasks have critical failures when attempting it whilst underskilled etc, and perhaps using stone should utilize this somehow. Due to stone being more readily available something like this would probably bring it infinitely closer to a more realistic situation of "oh damn, I just cleaved my stone in half... throw it in the pile and start again".

This is almost exactly what I was thinking of.

**Comment by NyarlathotepZERO** [comment]
What is the point of crafting a stone axe ?

**Comment by Shadefang** [comment]
It seems as if the point is (much as it would be in reality) to have an option that is available in the wilderness without relying on the RNG of foraging for metal.

**Comment by Night-Pryanik** [comment]
Closing as stale, since stalebot can't do this by itself.
